### ISSUE TITLE
Some questions about what ratification means for this standard

### ISSUE THREAD

**Issue by baskaufs** [tdwg/ggbn#3]
My questions about the proposed GGBN standard are related to how it will operate as a TDWG Standard. If I am understanding the situation correctly, the GGBN Data Standard is a standard that was created by an organization outside of TDWG (the Global Genome Biodiversity Network), and that standard is now being recommended for adoption by TDWG. This approach is not without precedent in TDWG. A large number of TDWG [Prior standards](https://www.tdwg.org/standards/) were developed by non-TDWG groups, then adopted by TDWG as a standard. However, there are no examples of that sort among the TDWG "Current standards", particularly among the vocabulary standards.

So that leads me to wonder about some issues related to maintenance of this standard. The existing precedent for Current Standards that are vocabularies (Darwin and Audubon Cores) involves the minting of some terms whose namespaces fall within in the `http://rs.tdwg.org/` subdomain. The terms in those namespaces "belong" to TDWG and the responsibility for maintenance of those terms falls to the vocabulary maintenance group chartered to maintain that particular vocabulary. There is a well-defined process, found in the [TDWG Vocabulary Maintenance Specification (VMS)](http://rs.tdwg.org/vms/doc/specification/), that lays out the criteria for making decisions about changes to existing terms and adding new terms in TDWG namespaces. 

Both Darwin and Audubon Cores "borrow" terms from other vocabularies whose IRIs fall within namespaces outside of the `http://rs.tdwg.org/`subdomain. In the case of Darwin Core, the small number of borrowed terms are from Dublin Core, and those terms are exceptionally stable. Audubon Core borrows a large number of terms from numerous other namespaces and exact procedure for handling changes in these borrowed terms as they change is not yet entirely clear to the Audubon Core Maintenance Group. Many of these borrowed terms are not managed by a standards organization, so changes in those terms can happen without warning and without public input. Should Audubon Core automatically update the versions of those terms whenever they change? The current [policy (decision 2.1)](https://github.com/tdwg/ac/blob/master/policies.md) of the Maintenance Group is that those versions NOT be updated automatically unless the Maintenance Group has assessed the likely effect of such changes on the stability and interoperability of Audubon Core. This has not yet been done for any of the existing borrowed terms, so they are effectively "frozen" at the versions which were initially borrowed. The situation is further complicated by the fact that term versioning is not clear in some of the vocabularies that serve as sources for the borrowed terms.

So my question is how this standard will be maintained if it were to be adopted. The terms in the proposed standard are in the `ggbn:` namespace (`http://data.ggbn.org/schemas/ggbn/terms/`), which is not within the TDWG-controlled `http://rs.tdwg.org/` subdomain. So for all practical purposes, the entire GGBN vocabulary is composed of "borrowed" terms. Presumably the maintenance of these terms falls to GGBN and not TDWG. If GGBN decides to change or add terms, what will be the effect on the actual TDWG standard? The reason for the adoption of the TDWG VMS was to eliminate the ambiguity surrounding the process of vocabulary changes and additions. If we simply say that TDWG is "ratifying" the GGBN standard, what does that mean about the change process for that vocabulary?  My impression that the GGBN vocabulary was developed by a community-based process. Is it GGBN's intention that this will continue, or is GGBN turning over the control of the vocabulary to TDWG? Will there be a GGBN Maintenance Group within TDWG that will serve as the gatekeeper for the changes to the GGBN standard that are actually "accepted" by TDWG? What happens if the GGBN community decides to change some terms, but the TDWG GGBN Maintenance Group rejects those changes based on the criteria set out in the VMS? 

I am not saying that the GGBN Data Standard should not be adopted as a TDWG standard. What I am saying is that it would be a really bad idea to adopt it without clear answers to these questions of vocabulary maintenance. These questions need to be addressed transparently and publicly before this proposal moves further through the ratification process.

I'm also concerned about the level of transparency in the review process itself. The [TDWG Process](https://www.tdwg.org/about/process/) which are official by-laws of the organization, designate that a Review Manager be appointed to manage the review process. The text of the process document do not specify the exact role that the Review Manager plays in the public review, but the flow chart document indicates that the Review Manager is responsible for managing the public review. In all recent standards ratifications, the person who has been the review manager has had a very public-facing role in managing the ratification process and has served independently from the Executive Committee in managing the review and making determinations about whether the public comments have been addressed by the Task Group proposing the draft. Who is the review manager for this ratification? For that matter, who are the members of the Task Group proposing the draft? The process specified in the Process document aren't suggestions, they are the actual by-laws of TDWG and it isn't clear to me that they are being followed here.

It is also the review manager's responsibility to make sure that the proposed standards documents conform to the [TDWG Standards Documentation Specification (SDS)](http://rs.tdwg.org/sds/doc/specification/) before they are submitted for ratification. The proposal should clearly designate specific documents that are to be included in the standard and each of those documents should be examined to make sure that they conform to the SDS. Based on the public review announcement, the "actual specification" is on the TDWG Terms Wiki, but the link I found was to the ggbn wiki. The description on the landing page says it describes "Draft Standard version 1" released on October 2016 but that a new stable version of the standard will be released at the end of 2019. What version are we actually adopting here? 

The document at https://wiki.ggbn.org/ggbn/GGBN_Data_Standard_v1 does not conform to the SDS as required. In particular, there is no indication of the Type of the terms (Class, Property, or Concept). Some terms in the vocabulary seem to be properties, while others appear to be controlled vocabulary terms (concepts). The authors need to go through this document and check for conformance to the SDS.

There is another linked document at https://wiki.ggbn.org/ggbn/Mandatory_and_recommended_fields_for_sharing_data_with_GGBN . Is this intended to be included as part of the standard? If so, it should have headers indicating this. There is also no indication of what parts of the document are normative, non-normative, etc. Again, if this document is intended to be part of the standard, it needs to conform to the SDS.

To summarize:
1. The responsible parties (composition of the task group and the name of the review manager) need to be made public.
2. This proposal needs to state clearly what documents are included in the standards proposal.
3. Each document designated as part of the proposed standard need to conform to the SDS.
4. There needs to be clarity about how this vocabulary will be maintained in a manner consistent with TDWG policy as laid out in the VMS.
